src/pyproject_hooks/_impl.py

```diff
@@ -132,7 +158,7 @@ def __init__(
         self.python_executable = python_executable
 
     @contextmanager
-    def subprocess_runner(self, runner):
+    def subprocess_runner(self, runner: "SubprocessRunner") -> Iterator[None]:
```

Comment:
```suggestion
    def subprocess_runner(self, runner: SubprocessRunner) -> Iterator[None]:
```

Reply:
It isn't for 3.8+, but we're preserving 3.7 compatibility (see below).